device/mellanox/x86_64-mlnx_msn4410-r0/ACS-MSN4410/qos.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/qos.json.j2
```

Comment:
are you sure? i prefer in any case it will be using the sn4700 even for visuality

Reply:
[ACS-MSN4700/qos.json.j2](https://github.com/Azure/sonic-buildimage/blob/master/device/mellanox/x86_64-mlnx_msn4700-r0/ACS-MSN4700/qos.json.j2)

It is the same as for SN4700, please check the link above